image/system_a/data/lang/English.json

```diff
@@ -234,9 +234,9 @@
   "sim_wrong_pin_confirmation": "Wrong PIN code.",
   "sim_pin_changed_successfully": "PIN code changed successfully",
   "sim_card_changed_successfully": "SIM card changed successfully.",
-  "sim_card_select_successfully": "<text>SIM select process<br></br>has been successfully.</text>",
+  "sim_card_select_successfully": "<text>SIM selection process has been<br></br>successfully completed.</text>",
```

Comment:
_successfully completed_ - kind of redundant ;) _successful_ or _completed_ would just suffice

Reply:
True, fixed ;)